packages/specifications/src/projet/lauréat/représentant-légal/stepDefinitions/représentantLégal.then.ts

```diff
@@ -36,23 +36,27 @@ Alors(
 Alors(
   `le représentant légal du projet lauréat( ne) devrait( pas) être mis à jour`,
   async function (this: PotentielWorld) {
-    await waitForExpect(async () => {
-      const { identifiantProjet } = this.lauréatWorld;
-
-      const représentantLégal =
-        await mediator.send<ReprésentantLégal.ConsulterReprésentantLégalQuery>({
-          type: 'Lauréat.ReprésentantLégal.Query.ConsulterReprésentantLégal',
-          data: {
-            identifiantProjet: identifiantProjet.formatter(),
-          },
-        });
-
-      const actual = mapToPlainObject(représentantLégal);
-      const expected = mapToPlainObject(
-        this.lauréatWorld.représentantLégalWorld.mapToExpected(identifiantProjet),
-      );
-
-      actual.should.be.deep.equal(expected);
-    });
+    try {
+      await waitForExpect(async () => {
+        const { identifiantProjet } = this.lauréatWorld;
+
+        const représentantLégal =
+          await mediator.send<ReprésentantLégal.ConsulterReprésentantLégalQuery>({
+            type: 'Lauréat.ReprésentantLégal.Query.ConsulterReprésentantLégal',
+            data: {
+              identifiantProjet: identifiantProjet.formatter(),
+            },
+          });
+
+        const actual = mapToPlainObject(représentantLégal);
+        const expected = mapToPlainObject(
+          this.lauréatWorld.représentantLégalWorld.mapToExpected(identifiantProjet),
+        );
+
+        actual.should.be.deep.equal(expected);
+      });
+    } catch (e) {
+      console.error('😮‍💨', e);
```

Comment:
ce try catch est voulu ou c'était juste du debug ?
```suggestion
      this.error = e;
```